packages/svelte/src/internal/client/runtime.js

```diff
@@ -1017,6 +1046,12 @@ export function set_signal_value(signal, value) {
 			}
 		}
 	}
+
+	// @ts-expect-error
+	if (DEV && signal.inspect) {
+		// @ts-expect-error
+		for (const fn of signal.inspect) fn();
```

Comment:
Don't you want to fire this within the if block above?

Reply:
oops, fixed